File: backend/ideappservice/src/main/java/com/ideapp/ideappservice/model/JoinedClub.java
package com.ideapp.ideappservice.model;

public class JoinedClub {
    String clubName;
    String parentClub;
    String imageUrl;
    String description;

    public JoinedClub(){}
    public JoinedClub(String clubName, String parentClub,String imageUrl, String description) {
        this.clubName = clubName;
        this.parentClub = parentClub;
        this.imageUrl = imageUrl;
        this.description = description;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }

    public String getClubName() {
        return clubName;
    }

    public void setClubName(String clubName) {
        this.clubName = clubName;
    }

    public String getParentClub() {
        return parentClub;
    }

    public void setParentClub(String parentClub) {
        this.parentClub = parentClub;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }
}

File: frontend/frontend_idea_app/lib/models/JoinedClub.dart
class JoinedClub {
  String clubName, parentClub, imageUrl, description;
  int vote;

  JoinedClub(this.clubName, this.parentClub, this.imageUrl, this.vote,
      this.description);

  JoinedClub.fromJson(Map<String, dynamic> json)
      : parentClub = json['parentClub'],
        imageUrl = json['imageUrl'],
        clubName = json['clubName'],
        vote = json['vote'],
        description = json['description'];

  Map<String, dynamic> toJson() => {
        "clubName": "'" + clubName + "'",
        "parentClub": "'" + parentClub + "'",
        "imageUrl": "'" + imageUrl + "'",
        "vote": vote,
        "description": "'" + description + "'",
      };
}